swagger.yml

```diff
@@ -76,7 +76,7 @@ paths:
         "405":
           description: "Method not allowed"
         "503":
-          $ref: '#/responses/InternalServerError'
+           description:  "User is not authorised to make the request"
```

Comment:
Please use the parameterised values here, there is no point repeating the messages again. The last block has the common 401, 403 and 503 parameterised